### PR TITLE
initialize Bookstore project with components

### PR DESCRIPTION
**Pull request by frempongdev**
### - Initialized Bookstore App with components. 🚀 
  - Installed React App Boilerplate with CRA.
  - Added components directory. 📁 
  - Installed `React-Router-Dom`.
  - Setup routers.
  - Added Books and Categories Routes.
  - Added `Book.js` component.
  - Updated `Book.js` component to display book title, book author and a "Remove" `<button>`.
  - Added `Book.js` to `Books.js` as child that takes title and author props.
  - Added a `<Form>` component with title and author inputs to `Books.js` component. 🧾
  - Added "Check Status" `<button>` to Categories Route. 
  - Added style directory with `Books.css`, `Book.css` and `Naviigation.css` to be imported for minimal style. 📁 
  - Fixed Linter errors. 🧰

### CODE REVIEW

File: src/components/books.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        <li><Book title="Capital in the 21st Century" author="Suzanne Collins" /></li>
+      </ul>
+    </div>
+    <div>
+      <form>
+        <h3>ADD NEW BOOK</h3>
+        <input placeholder="Book Title" type="input" />
+        <input placeholder="Author" type="input" />
+        <input type="submit" value="Add Book" />
+      </form>
```
Comment (ichala) [inline src/components/books.js]: - [x] as part of the requirements the Book form should be a component by itself and be called in  book.js as `<NewBookForm>` , kindly fix that 🙏

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 
- React Installed ✔️ 
- created components✔️ 


 ## Required Changes ♻️


- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/books.js (hunk 1 above)
Comment (frempongdev) [reply src/components/books.js]: Alright.
Thanks for pointing this out to me. Fixing it now.

Comment (rloterh) [reply src/components/books.js]: resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Frempong,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉 💯 🌟 

## Highlights 👍
- Good job implementing requested changes!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
- Kindly include instructions under the _Getting Started_ section in your ReadMe, on how to **install** and **run** your application. Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by frempongdev** [comment]
Thanks to @ichala and @rloterh for going through this Milestone of my project.

**Pull request merged**